portfolio/src/main/webapp/index.html

```diff
@@ -1,56 +1,54 @@
 <!DOCTYPE html>
-<html>
+<html lang="en">
   <head>
-    <meta charset="UTF-8">
+    s<meta charset="UTF-8">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-        <div id = "nav_bar">
+        <div id = "nav-bar">
```

Comment:
Spacing here is inconsistent.

```suggestion
        <div id="nav-bar">
```